parachain/pallets/omni-account/src/lib.rs

```diff
@@ -171,7 +172,7 @@ pub mod pallet {
 				who: omni_account,
 				result: result.map(|_| ()).map_err(|e| e.error),
 			});
-			Ok(())
+			Ok(Pays::No.into())
```

Comment:
I've also noticed it missing but in my case I had to:
https://github.com/litentry/litentry-parachain/pull/3132/files#diff-960f61f16faf996e829d1decb06b54fb404525cc02cdc585b9e8db86d596014dR344

Otherwise all extrinsincs signed by newly generated account were rejected (low funds)